package.json

```diff
@@ -35,18 +36,18 @@
     "dev:api": "ts-node test/server/server.ts",
     "dev:background": "yarn run dev:clean && pm2 start ecosystem.config.js",
     "docker": "docker build -t api3/airkeeper .",
+    "export-aws-env": "test -f config/aws.env && export $(egrep -v '^#' config/aws.env | xargs)",
     "lint": "yarn run prettier:check && yarn run lint:eslint",
     "lint:eslint": "eslint . --ext .js,.ts",
     "prettier:check": "prettier --check \"./**/*.{js,sol,ts,md,yml,json}\"",
     "prettier:write": "prettier --write \"./**/*.{js,sol,ts,md,yml,json}\" --ignore-unknown",
-    "sls:config": "sls config credentials --provider aws --key $AWS_ACCESS_KEY_ID --secret $AWS_SECRET_ACCESS_KEY",
-    "sls:deploy": "yarn build && sls deploy --region $REGION --stage $STAGE",
-    "sls:remove": "sls remove --region $REGION --stage $STAGE",
+    "sls:deploy": "sls deploy",
+    "sls:remove": "sls remove",
     "sls:invoke:rrp-beacon-update": "sls invoke --function rrp-beacon-update",
-    "sls:invoke-local:rrp-beacon-update": "yarn build && sls invoke local --function rrp-beacon-update",
+    "sls:invoke-local:rrp-beacon-update": "sls invoke local --function rrp-beacon-update",
```

Comment:
Why did you remove the `build` part? Don't you need to build the lambda before invoking it?

Reply:
yes, also good catch! I was playing around with `pre` and `post` scripts. I added a `predeploy:sls` script where I was planning to do the `yarn build` but then I didn't like the `pre` script name being so long for running the functions locally so I dropped the idea but forgot to add the `yarn build &&` back.